GitCommands/RevisionReader.cs

```diff
@@ -97,6 +97,30 @@ public IReadOnlyCollection<GitRevision> GetRevisionsFromList(IList<ObjectId> unt
             return GetRevisionsFromArguments(arguments, cancellationToken);
         }
 
+        /// <summary>
+        /// Get the GitRevisions for all commits between 2 commits (boundaries included).
+        /// </summary>
+        /// <param name="olderCommitId">older commit Id.</param>
+        /// <param name="newerCommitId">newer commit Id.</param>
+        /// <param name="cancellationToken">Cancellation cancellationToken.</param>
+        /// <returns>List with GitRevisions.</returns>
```

Comment:
```suggestion
        /// <summary>
        ///  Retrieves the git history in the requested range (boundaries included).
        /// </summary>
        /// <param name="olderCommitId">The first (older) commit.</param>
        /// <param name="newerCommitId">The last (newer) commit.</param>
        /// <param name="cancellationToken">A cancellation token.</param>
        /// <returns>The retrieved git history.</returns>
```

I'm a little confused with the terminology - are we using `commit`, `commit hash`, `commit id`, or something else...?

Reply:
Thanks for your comment that makes me investigate.
From what I can get looking at GE source code, `commit id` is used when the type is `ObjectId`. When it is the string, it is `commit hash`. So I will rename the parameters.